frontend/src/features/public-form/components/FormPaymentPage/components/PaymentCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Flex, Text, useDisclosure } from '@chakra-ui/react'
+
+import { ProductItem } from '~shared/types'
+import { centsToDollars, formatCurrency } from '~shared/utils/payments'
+
+import { BxsChevronDown } from '~assets/icons/BxsChevronDown'
+import Button from '~components/Button'
+import Checkbox from '~components/Checkbox'
+import Radio from '~components/Radio'
+
+import PaymentQuantityModal from './PaymentQuantityModal'
+
+const ItemQuantityButton = ({
+  product,
+  onClick,
+}: {
+  product: ProductItem
+  onClick: () => void
+}) => {
+  if (!product.data.multi_qty) {
+    return <></>
+  }
+  return (
+    <Box minWidth="6.75rem" height="2.75rem" onClick={onClick}>
+      <Button
+        rightIcon={<BxsChevronDown fontSize="1.5rem" />}
+        colorScheme="secondary"
+        aria-label="Change"
+        variant="clear"
+      >
+        Qty: {product.quantity}
+      </Button>
+    </Box>
+  )
+}
+
+export const PaymentCard = ({
+  product,
+  onItemChange,
+  isMultiSelect,
+}: {
```

Comment:
add `colorTheme` too? right now the colour of this component is not consistent with the colour theme (if the theme is not blue)

Reply:
Thanks for highlighting this! Addressed in 13ddb8b02

I'll let design side know about the theming issue cause we previously wanted to hardcode the values